library/core/src/ops/unsize.rs

```diff
@@ -33,40 +33,67 @@ use crate::marker::Unsize;
 /// [nomicon-coerce]: ../../nomicon/coercions.html
 #[unstable(feature = "coerce_unsized", issue = "18598")]
 #[lang = "coerce_unsized"]
-pub trait CoerceUnsized<T: ?Sized> {
+pub trait CoerceUnsized<T: ?Sized + PointeeSized> {
```

Comment:
After a bootstrap bump we'll be able to remove most `?Sized` from libcore, right?

Reply:
Yeah, they're just there because the bootstrap compiler doesn't understand that `PointeeSized` does a relxation.